### PR TITLE
Remove observation_adapter, reward_adapter, info_adapter from HiWayEnv

### PR DESCRIPTION
**Pull request by Adaickalavan**
Removes observation_adapter, reward_adapter, and info_adapter, from `HiWayEnv`.

See issue: #1174

### CODE REVIEW

File: examples/rl/rllib/rllib.py [hunk 1]
```diff
@@ -193,15 +194,15 @@ def main(
...
-        "scenario",
-        help="Scenario to run (see scenarios/ for some samples you can use)",
+        "--scenario",
         type=str,
+        default=str(Path(__file__).resolve().parents[3] / "scenarios/sumo/loop"),
+        help="Scenario to run (see scenarios/ for some samples you can use)",
     )
     parser.add_argument(
-        "--headless",
+        "--head",
```
Comment (Gamenot) [inline examples/rl/rllib/rllib.py:203]: Maybe `--envision` instead?

Comment (Adaickalavan) [reply examples/rl/rllib/rllib.py]: Changed `--head` to `--envision`.

File: smarts/zoo/agent_spec.py [hunk 2]
```diff
@@ -67,13 +67,9 @@ class AgentSpec:
     """Parameters to be given to `AgentSpec.agent_builder` (default None)"""
 
     observation_adapter: Callable = lambda obs: obs
-    """An adaptor that allows shaping of the observations (default lambda obs: obs)"""
     action_adapter: Callable = lambda act: act
-    """An adaptor that allows shaping of the action (default lambda act: act)"""
     reward_adapter: Callable = lambda obs, reward: reward
-    """An adaptor that allows shaping of the reward (default lambda obs, reward: reward)"""
     info_adapter: Callable = lambda obs, reward, info: info
```
Comment (Gamenot) [inline smarts/zoo/agent_spec.py:72]: I think we should add deprecation warnings about these.

**Review by Gamenot — APPROVED** [review]

File: smarts/zoo/agent_spec.py (hunk 2 above)
Comment (Adaickalavan) [reply smarts/zoo/agent_spec.py]: Added a deprecation warning message.

**Pull request merged**